WcaOnRails/db/migrate/20170417072301_remove_description_from_teams.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class RemoveDescriptionFromTeams < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :teams, :description, :text
+    remove_column :teams, :name, :string
+    add_column :teams, :email, :string
+    add_column :teams, :rank, :integer
```

Comment:
The only problem with the `rank` column solution is that now you have to validate it. At least for being unique.

I'd also call it `display_rank`.